IMPACT-ASSESSMENTS.md

```diff
@@ -0,0 +1,10 @@
+# Impact assessments
+
+To provide a clearer picture of the downstream effects of a proposal, the CLC requires that proposers draft an impact assessment of their changes before a final decision is reached. An impact assessment MUST be completed if the proposal contains [breaking changes](https://pvp.haskell.org/), and is RECOMMENDED even if the changes are simply additive, especially in cases of anticipated name clashing or when re-exporting from the Prelude.
+
+## The process
+
+- Fork and build [a newer version of GHC](https://gitlab.haskell.org/ghc/ghc#building-installing) (say, the latest official release) that includes the changes made in your proposal. You should also make a merge request with your changes and add it to your proposal's issue.
```

Comment:
I don't understand "a newer version of GHC" and "the latest official release". Taken literally, they seem contradictory to me. Does it mean "take the tagged commit of the latest official GHC release and add to it just the change you are proposing to make"?

Reply:
Honestly I'm not super clear on the details here. You're correct that that was the meaning I was trying to convey; whether that's "best practice" is another question!